server-worker/src/main/scala/com/aliyun/emr/rss/service/deploy/worker/Worker.scala

```diff
@@ -186,19 +186,24 @@ private[deploy] class Worker(
   }
 
   def heartBeatToMaster(): Unit = {
-    val shuffleKeys = new jHashSet[String]
-    shuffleKeys.addAll(partitionLocationInfo.shuffleKeySet)
-    shuffleKeys.addAll(localStorageManager.shuffleKeySet())
+    val shuffleResourceConsumption = new jHashMap[String, java.lang.Long]
+    partitionLocationInfo.shuffleKeySet.asScala.foreach { shuffleKey =>
+      shuffleResourceConsumption.put(shuffleKey, 0)
```

Comment:
Is it necessary to initial all shuffleKeys to 0?

Reply:
I think it's necessary because partitionLocationInfo has no disk usage information.